modules/ballerina-http/src/main/ballerina/ballerina/net/http/natives.bal

```diff
@@ -1,16 +1,36 @@
 package ballerina.net.http;
 
-@Description { value:"Represents an HTTP request message"}
-@Field {value:"host: The server host name"}
+@Description { value:"Represents the HTTP server connection"}
```

Comment:
Again, is it correct to refer to this Connection struct as an "HTTP server connection"?

Reply:
Changed it as HTTP server connector connection